reporter/statsd/statsd.go

```diff
@@ -0,0 +1,83 @@
+// Package statsd implements an statsd client.
+package statsd
+
+import (
+	"time"
+
+	"github.com/cactus/go-statsd-client/v5/statsd"
+)
+
+// Option represents statsd option function.
+type Option func(*Statsd)
+
+// WithFlushInterval sets the maximum flushInterval for packet sending.
+// Defaults to 300ms.
+func WithFlushInterval(interval time.Duration) Option {
+	return func(s *Statsd) {
+		s.flushInterval = interval
+	}
+}
+
+// WithFlushBytes sets the maximum udp packet size that will be sent.
+// Defaults to 1432 flushBytes.
```

Comment:
Where do those default values come from? I don't see a `DefaultConfig` struct or anything like that anywhere. If those default values are referenced in the package, they shouldn't be set externally IMO but be part of the package as well. I assume that since I can't see them they must come from the `cmd`? If so, I'd recommend having a `DefaultConfig` in this package which the `cmd` can override when given specific values. WDYT?

Reply:
Good catch, must be an old comment.